ui/components/app/detected-token/detected-token-aggregators/index.scss

```diff
@@ -0,0 +1,13 @@
+.detected-token-aggregators {
+  .typography {
+    display: inline;
+  }
+
+  & &__link {
```

Comment:
Is there a way to avoid the double `& &` here?

Reply:
Agree, we should use this only if there are specificity issues. Hoping to get to the swapping of the ui and app stylesheets at some stage. One line change but needs A LOT of testing.

Reply:
Without `& &` the styles of the <Button/> component is overriding the styles needed for this link.

Reply:
Ok, no problem.